app/src/main/java/ch/epfl/sweng/rps/LoadingActivity.kt

```diff
@@ -0,0 +1,46 @@
+package ch.epfl.sweng.rps
```

Comment:
Could you explain more about this class, what is this used for?

Reply:
Basically, for different reasons (using firebase emulators etc) I had to disable Firebase automatic initializing. It thus had to be done somewhere, here. Also other logic can happen in the future (everything needing initialization)

Reply:
Thanks for your explaination! Take care and have a rest bro :)